src/Core/src/Platform/Windows/MauiWebView.cs

```diff
@@ -157,5 +170,19 @@ void SetupPlatformEvents()
 				}
 			};
 		}
+
+		static bool IsUriWithLocalScheme(string? uri)
+		{
+			return uri?
+				.ToLowerInvariant()
+				.StartsWith(
+					LocalScheme.TrimEnd('/').ToLowerInvariant()) == true;
+		}
+
+		static bool IsDataUri(string? uri)
+		{
+			return uri?
+				.StartsWith("data:text/html") ?? false;
```

Comment:
Similar to the above comment, change to:

```suggestion
			return uri?.StartsWith("data:text/html", StringComparison.Ordinal) ?? false;
```

Reply:
I'll make that change, too

Reply:
I pushed a changeset with the requested changes.